spec/characterization_services/mediainfo_characterization_service_spec.rb

```diff
@@ -121,13 +122,14 @@
       allow(audio_track_attributes).to receive(:encoded_date).and_return Time.zone.parse("UTC 2009-03-30 19:49:13")
       allow(audio_track_attributes).to receive(:producer).and_return("Test Producer")
       allow(audio_track_attributes).to receive(:originalsourceform).and_return("cassette")
-      allow(audio_track_attributes).to receive(:duration).and_return(0.261)
+      allow(audio_track_attributes).to receive(:duration).and_return(261)
```

Comment:
@eliotjordan this looks like a unit change? Do we need a migration to go with this?